src/index.js

```diff
@@ -2,16 +2,14 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
 
```

Comment:
- [ ] It would be better if we update our README file to make it look a little more professional and project-specific. It would be better to include the  deployed version of the app (Heroku) or other hosting site in the README